### ISSUE TITLE
Compress appears to break

### ISSUE THREAD

**Issue by aaruni96** [archiecobbs/s3backer#149]
I am on the latest commit (2712cd70), and after updating to it, I run into a fuse error if I try to invoke s3backer with the compress flag.

Here is the exact command I'm trying to run : 

```
sudo s3backer bucket --prefix=myprefix/ --accessFile=/path-to-passwd-file --baseURL='https://s3.eu-central-1.wasabisys.com/' --blockSize=128k --size=102400G s3-b --compress --listBlocks --blockCacheFile=.blockCache --blockCacheRecoverDirtyBlocks --blockCacheSize=40960 --blockCacheWriteDelay=5000
```

And the exact output (and error) is like here : 

```
s3backer: auto-detecting block size and total file size...
s3backer: auto-detected block size=128k and total size=100t
fuse: unknown option `--compress`
```

The exact same command used to work in a previous version. Not fully sure what version it was, but my best guess is it was around commit number 467e507.

**Comment by aaruni96** [comment]
`--compress` works fine even on ef5fbbd . Probably something broke or change during the refactor for multiple compression algorithms.

Glancing through the changes, it looks like --compress isn't supported on its own, but one must now define what algorithm to use? I'm not sure if I'm right, and I can't figure out what the valid options or the format is.

**Comment by archiecobbs** [comment]
Should be fixed in 7cd192c.
